server/src/services/firewood-permit-service.es6

```diff
@@ -156,6 +156,16 @@ firewoodPermitService.sendEmail = (savedPermit, permitPng, rulesHtml, rulesText)
   return email.sendEmail('christmasTreesPermitCreated', savedPermit, attachments);
 };
 
+firewoodPermitService.emailPDF = (data) => {
+  const emailData = data;
+  emailData.attachments = [{
+    filename: 'permit.pdf',
+    content: Buffer.from(emailData.permitHTML, 'utf-8'),
+    contentType: 'application/pdf'
```

Comment:
Is the HTML being converted to a PDF somewhere along the line? Not sure if that's something Nodemailer knows how to do, or if it's implemented elsewhere in here and I haven't seen it.

In any case, just wanted to clarify because I see three possibilities:
1. this is a placeholder until PDF creation is ready
2. PDF creation is happening somewhere and I didn't see it
3. this is a bug

I figure it's one of the first two (I see an empty `pdf.service.es6`, for example), but... it wouldn't be a good review if I didn't point and ask. 😛